pkg/parser/mysql/const.go

```diff
@@ -191,6 +191,9 @@ const (
 const (
 	// SystemDB is the name of system database.
 	SystemDB = "mysql"
+	// SysDB is the name of `sys` schema, which is a set of objects to help users to interpret data collected
+	// in `information_schema`.
+	SysDB = "sys"
```

Comment:
It's a bad name, and it's especially confusing with the `SystemDB` variable above 🤦 . Does anyone has better idea on the variable name?

Reply:
clould we rename `SystemDB ` to `MysqlDB` ? BTW, the method `util.IsSysDB` and `isSystemTable` seems also confusing may be we can remove `util.IsSysDB` nd rename `isSystemTable` to `undropptableSysTable` ?

Reply:
> clould we rename `SystemDB` to `MysqlDB` ? 

I agree with you 👍 . I'll do this modification in a separated PR, so that it'll not make this one harder to read.

> may be we can remove `util.IsSysDB`

There are still some `*SysDB` functions like `IsMemOrSysDB` and `hasSysDB`, which are more widely used and call the `IsSysDB`🤔 . I don't think removing `util.IsSysDB` and inline the logic inside these function is better.

Maybe we can reach an agreement on the term at first and do some modification in the next PR (with `MysqlDB`):

`SysDB` means `sys`, `SystemDB` means `mysql`, `sys`, `information_schema` and other databases introduced by `TiDB` itself. `SystemTable` and `SysTable` means all tables in the `SystemDB`.

If so, we'll need to rename these function into `IsSystemDB`, `IsMemOrSystemDB` and `hasSystemDB`.

>  rename `isSystemTable` to `undropptableSysTable`

Good suggestion. Done. It has been modified to `isUndroppableTable` 👍 .